runtime/src/iree/hal/utils/CMakeLists.txt

```diff
@@ -238,4 +238,23 @@ iree_cc_test(
     iree::testing::gtest_main
 )
 
+iree_cc_library(
+  NAME
+    deferred_work_queue
+  HDRS
+    "deferred_work_queue.h"
+  SRCS
+    "deferred_work_queue.c"
+  DEPS
+    ::deferred_command_buffer
+    ::resource_set
+    ::semaphore_base
+    iree::base
+    iree::base::internal::arena
+    iree::base::internal::synchronization
+    iree::base::internal::threading
+    iree::hal
+  PUBLIC
+)
```

Comment:
Should this be wrapped in:
```
if (IREE_ENABLE_THREADING)
```

My builds is now failing as `iree::base::internal::threading` does not exist, presumably because `IREE_ENABLE_THREADING` is not set:

https://github.com/iree-org/iree/blob/b6602e853ceabe325ecb6d0336e747198e647a8c/runtime/src/iree/base/internal/CMakeLists.txt#L383-L406

Reply:
https://github.com/iree-org/iree/pull/18234